### PR TITLE
network - increase ping checks from 20 - 40

### PR DESCRIPTION
**Pull request by yuval-lb**
we see cases where the link is setup only after 22-25 ping checks

### CODE REVIEW

**Comment by sashas-lb** [comment]
Did you check what is the reason 1 minute for boot seems harsh

**Comment by yuval-lb** [comment]
dmesg didnt show anything special other then the link went up after 82 seconds. what else I can check?

**Comment by sashas-lb** [comment]
journal .. when network service activated and why

**Comment by yuval-lb** [comment]
i dont think we have journal in the inaugurator

File: inaugurator/network.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 class Network:
     _CONFIG_SCRIPT_PATH = "/etc/udhcp_script.sh"
-    _NR_PING_ATTEMPTS = 20
+    _NR_PING_ATTEMPTS = 40
```
Comment (sashas-lb) [inline inaugurator/network.py]: oh wow .. it is in inaugurator .. how did we get to that ?

**Comment by sashas-lb** [comment]
oh WOW i though it is in rootfs stage .. it took 40 seconds to get a ping
.. fuck .. what did we do ?

**Comment by yuval-lb** [comment]
all the selftest is happening after this ping check finish.
only thing I have in mind  - is in the init file we added load nvme. but this machine dont have nvme devices.
but it looks like the network driver which is taking so much time.

**Pull request merged**